tests/capture/test_nested_plxpr.py

```diff
@@ -154,6 +154,7 @@ def qfunc(wire):  # x is closure variable and a tracer
         assert len(q) == 1
         qml.assert_equal(q.queue[0], qml.adjoint(qml.RX(2.5, 2)))
 
+    @pytest.mark.xfail(raises=NotImplementedError)
```

Comment:
Why again? :see_no_evil:

Reply:
Once I add #6343, I'll be able to capture the operations from the inner jaxpr, and then be able to apply the operations in reverse. Right now, DQ doesn't support this.

Reply:
Understood, thanks!